lib/OnUserTypingHandler.ts

```diff
@@ -0,0 +1,61 @@
+import { IHttp, IPersistence, IRead } from '@rocket.chat/apps-engine/definition/accessors';
+import { IApp } from '@rocket.chat/apps-engine/definition/IApp';
+import { ILivechatRoom, IVisitor } from '@rocket.chat/apps-engine/definition/livechat';
+import { RocketChatAssociationModel, RocketChatAssociationRecord } from '@rocket.chat/apps-engine/definition/metadata';
+import { IRoomUserTypingContext } from '@rocket.chat/apps-engine/definition/rooms';
+import { AppSettingId } from '../enum/AppSettingId';
+import { ErrorLogs } from '../enum/ErrorLogs';
+import { retrievePersistentTokens } from '../helperFunctions/PersistenceHelpers';
+import { chasitorTyping } from '../helperFunctions/SalesforceAPIHelpers';
+
+export class OnUserTypingHandler {
+	constructor(
+		private app: IApp,
+		private data: IRoomUserTypingContext,
+		private read: IRead,
+		private http: IHttp,
+		private persistence: IPersistence,
+	) {}
+
+	public async exec() {
+		this.app.getLogger().log(this.data);
```

Comment:
I would suggest using a normal console.log(), with the following structure similar to this: console.log(Infolog.RECIEVED_TYPING_HANDLER_DATA, data)

Reply:
It it not needed actually. I added it for debugging purpose and forgot to remove 😅.  Will remove it. 
Thanks 😄